process/lifecycle.md

```diff
@@ -115,87 +82,94 @@ To be accepted at the Incubation stage, a project must meet the Sandbox requirem
     *   A CODE_OF_CONDUCT file that sets the ground rules for participants’ behavior associated and helps to facilitate a friendly, welcoming environment. By default, projects should leverage the Linux Foundation Code of Conduct unless an alternate Code of Conduct was previously approved.
-    *   A RELEASE file that provides documentation on the release methodology, cadence, criteria, etc.
+    *   A RELEASE file that documents the release methodology, cadence, criteria, etc.
     *   A GOVERNANCE file that documents the project’s technical governance.
     *   A SUPPORT file to let users and developers know about ways to get help with your project.
-*   Have achieved and maintained a OpenSSF Best Practices Badge at the [passing level](https://bestpractices.coreinfrastructure.org/en/criteria). 
+*   Have achieved and maintained an OpenSSF Best Practices Badge at the [passing level](https://bestpractices.coreinfrastructure.org/en/criteria). 
 *   Have had a successful license scan with any critical issues remedied.
 *   Have a defined project mission and scope
 *   An overview of the project’s architecture and features defined.
 *   A project roadmap defined, which should address the following questions.
     *   What use cases are possible now?
-    *   What does the next year look like in terms of additional features and use cases covered?
+    *   What does the next year look like regarding additional features and use cases covered?
 *   Community and contributor growth assessment
-    *   The current number of contributors and committers, and the number of different organizations contributing to the project. 
+    *   The current number of contributors, committers, and different organizations contributing to the project. 
     *   Demonstrate a sustained flow of commits / merged contributions
-    *   A credible plan for developing a thriving user community, in particular expanding the number of committers and contributors?
-    *   Outline of the plan for the project to complete the requirements for Adopted Stage
+    *   A credible plan for developing a thriving user community, particularly expanding the number of committers and contributors?
+    *   Outline of the plan for the project to complete the requirements for the [Adopted Stage](#adopted-stage)
 * Obtain an affirmative vote of the TAC.
 
-Sandbox Projects may propose to be reviewed to move to the Incubation Stage at any time by adding the review to a future TAC meeting agenda ( minimum two weeks notice required ) or may be moved to the Incubation Stage during its annual review. Projects should prepare a presentation outlining how it has completed the Incubation Stage requirements.
+Projects preparing to be considered for the Incubation Stage should prepare a presentation outlining how it has completed the requirements and present it at the TAC meeting where it is being considered.
```

Comment:
Should this say something about scope / duration / format of said presentation?

Reply:
I reworked this to point to the Annual Review format, just to make it easier